tests/PushServiceTest.php

```diff
@@ -42,7 +42,7 @@ public static function setUpBeforeClass(): void
      */
     protected function setUp(): void
     {
-        if (!(getenv('TRAVIS') || getenv('CI'))) {
+        if (getenv('TRAVIS') || getenv('CI')) {
```

Comment:
Can we remove that altogether? The description below is wrong, this should be run on CI (these are the tests that really matter actually)

Reply:
The description is wrong, yes. But the test itself has some issues running, e.g.
https://github.com/marc1706/web-push-php/runs/2808589708?check_suite_focus=true

Reply:
Yes, that's why the tests are "broken" right now, and this is what needs to be fixed

Reply:
To clarify, the tests are correct, this is a problem with web-push-testing-service

Reply:
web-push-testing-service seems to be no longer maintained and has been archived: https://github.com/GoogleChromeLabs/web-push-testing-service
I've also tested around with it a bit more and even after updating it to newer dependencies it's inherently unstable and fails very often. Unless there is another way of having some form of functional testing I would actually recommend to remove these.